.bithoundrc

```diff
@@ -0,0 +1,13 @@
+{
+  "critics": {
+    "lint": {
+      "engine": "eslint"
```

Comment:
Since we use default filenames and placements for our `eslint` configs, bitHound already correctly detects that this project uses `eslint`, so there is no need to add this explicitly.